tensorrt_llm/_torch/models/modeling_phi4mm.py

```
@@ -413,18 +58,37 @@
+            self.device)
+        # Required by Phi4MMImageAudioEmbedding.
+        # See link: https://huggingface.co/microsoft/Phi-4-multimodal-instruct/blob/main/modeling_phi4mm.py#L701
+        self.phi4mm_wte = ref_phi4mm_model.model.embed_tokens.to(self.device)
+
     @torch.inference_mode()
     def __call__(
         self, inputs: TextPrompt, sampling_params: SamplingParams
     ) -> Tuple[List[int], Optional[ExtraProcessedInputs]]:
-        text_prompt, mm_data = inputs.get("prompt"), inputs.get(
-            "multi_modal_data", {})
+        text_prompt, mm_data, mm_processor_kwargs = inputs.get("prompt"), \
```

Ruff (0.12.2)

Local variable `mm_processor_kwargs` is assigned to but never used

Remove assignment to unused variable `mm_processor_kwargs`

(F841)